index.js

```diff
@@ -278,6 +278,15 @@ exports.build = function build(argv, callback) {
   }
 
   function doNpmPack(_, callback) {
+    var findPackPath = function(output) {
```

Comment:
The prevailing style closures in this module is to use named function statements and to put them at the end of the parent function, not assigned anonymous function expressions.

In this case, however, it is not even a closure and it is only used once, so I think it might be better off to just inline the checks than to make them a function.